packages/postgres-database/src/simcore_postgres_database/models/resource_tracker_service_runs.py

```diff
@@ -117,9 +117,40 @@ class ResourceTrackerServiceRunStatus(str, enum.Enum):
     sa.Column(
         "node_name",
         sa.String,
-        nullable=True,
+        nullable=False,
         doc="we want to store the node/service name/label for tracking/billing purposes and be sure it stays there even when the node is deleted.",
     ),
+    # Project/Node parent fields
+    sa.Column(
+        "parent_project_id",  # UUID
+        sa.String,
+        nullable=False,
+        doc="We want to store the parent project id for tracking/billing purposes and be sure it stays there even when the node is deleted (that's also reason why we do not introduce foreign key)",
+    ),
+    sa.Column(
+        "root_parent_project_id",  # UUID
+        sa.String,
+        nullable=False,
+        doc="we want to store the root parent project id for tracking/billing purposes and be sure it stays there even when the node is deleted.",
+    ),
+    sa.Column(
+        "root_parent_project_name",
+        sa.String,
+        nullable=False,
+        doc="We want to store the root parent project name for tracking/billing purposes and be sure it stays there even when the node is deleted (that's also reason why we do not introduce foreign key)",
+    ),
+    sa.Column(
+        "parent_node_id",  # UUID
+        sa.String,
+        nullable=False,
+        doc="we want to store the parent node/service id for tracking/billing purposes and be sure it stays there even when the node is deleted.",
+    ),
+    sa.Column(
+        "root_parent_node_id",  # UUID
```

Comment:
so here you do not need the name?

Reply:
no, I think the root/parent node name will not be used, so I didn't add it for now.